docs/components/eyebrow/index.html

```diff
@@ -0,0 +1,104 @@
+---
+title: Eyebrow
+minver: 0.24.0
+also:
+  components/eyebrow/test.html: Testing - Eyebrow
+  elements/hx-eyebrow-control: <hx-eyebrow-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    The {{page.title}} provides access to platform-level services and aggregate information
+     across a user's accounts at Rackspace.
+
+  </p>
+{% endblock %}
+
+{% block content %}
+
+<section>
+  <header>
+    <h2 id="grouped-menu">Nav Menu</h2>
+  </header>
+
+  <div>
+    <div class="navbar">
+      <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+      <div class = "right-menu">
+        <a href="#">
```

Comment:
### Please update your demo to use the sample icons/menu options below:

<img width="339" alt="Screen Shot 2020-07-15 at 3 35 34 PM" src="https://user-images.githubusercontent.com/10120600/87988295-b5fea300-caa5-11ea-9ebf-d37be22341bb.png">

Reply:
Changes have been done, Please review